CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Release Versions:
   and to get the index of the joint by its name (#210)
 - Add possibility to have a ssh server for Python testing (#211)
 - Add Python bindings for Parameter class
+- Add Python bindings for clproto encode / decode of Parameter class (#214)
```

Comment:
I was actually going to change it and then I thought it was probably going to be fixed in your PR #213. Since it isn't fixed there then yes I will do it!